ethereum/core/src/main/java/tech/pegasys/teku/core/ForkChoiceUtil.java

```diff
@@ -193,7 +193,7 @@ private static boolean should_update_justified_checkpoint(
    *     <a>https://github.com/ethereum/eth2.0-specs/blob/v0.8.1/specs/core/0_fork-choice.md#on_tick</a>
    */
   public static void on_tick(MutableStore store, UInt64 time) {
-    if (store.getGenesisTime().isGreaterThan(time)) {
```

Comment:
Should we still not keep the property of not executing `on_tick` before genesis time?

Reply:
Feels a bit defensive since time should be >= genesisTime, but added this back in for good measure